.release-please-manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "cli": "1.3.0-pre.1",
+  "cli": "0.9.9",
```

Comment:
This tells release please the current version, so with a breaking change the next one should be `v1.0.0`

Reply:
Don't our breaking changes usually get their minor version bumped? Or how does it work now?

Reply:
Yes we did than until https://github.com/cloudquery/cloudquery/pull/2050/files

Reply:
Ah cool, thanks 👍